lib/dor/workflow/client.rb

```diff
@@ -25,7 +25,7 @@ module Workflow
     # Create and update workflows
     class Client
       # From workflow-server-rails' app/models/workflow_step.rb
-      VALID_STATUS = %w[waiting completed error queued skipped started].freeze
+      VALID_STATUS = %w[waiting completed error queued skipped started retrying].freeze
```

Comment:
I think we need to change this too: https://github.com/sul-dlss/workflow-server-rails/blob/main/app/models/workflow_step.rb#L9 ?

Reply:
Yes, that will be in a forthcoming PR.